### DISCUSSION TITLE
Possible to boot without Dev mode?

### DISCUSSION THREAD

**Discussion by s1618** [Maccraft123/Cadmium#128] [General]
After installing cadmium on a Lenovo Duet 5, it boots without problem in dev mode on the emmc.
Anyone knows how to skip the dev mode prompt when booting? or even disable dev mode?
When I try to disable it, it crashes because of "OS kernel or rootfs failed signature check"

**Comment by chinh4** [comment]
I'm pretty sure you can't

**Comment by LoganMD** [comment]
this may or may not be in the works, but no promises

**Comment by petm5** [comment]
This is not currently possible because the Cadmium kernel is not signed by Google (and probably never will be.) Booting Cadmium without developer mode would probably involve replacing the signing keys in the firmware with our own keys.

**Reply by snguyenVention** [reply]
thanks for the reply
